packages/react-core/src/layouts/Grid/Grid.js

```diff
@@ -3,31 +3,63 @@ import PropTypes from 'prop-types';
 import styles from '@patternfly/patternfly-next/layouts/Grid/styles.css';
 import { css } from '@patternfly/react-styles';
 import { GutterSize, getGutterModifier } from '../../styles/gutters';
+import { getGridSpanModifier, gridSpans } from './gridUtils';
+import { DeviceSizes } from '../../styles/sizes';
+
+const gridItemSpanValueShape = PropTypes.oneOf(gridSpans);
 
 const propTypes = {
   children: PropTypes.any,
   className: PropTypes.string,
-  gutter: PropTypes.oneOf(Object.keys(GutterSize))
+  gutter: PropTypes.oneOf(Object.keys(GutterSize)),
+  span: gridItemSpanValueShape,
+  xs: gridItemSpanValueShape,
```

Comment:
xs actually no longer exists anymore as of `@patternfly/patternfly-next@v1.0.15` master has this removed.  You should only need to remove this from here and default props though.  Everything else should be taken care of.